.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 build
 .envrc
+node.env
```

Comment:
why do we need two env files? Shouldn't we just have `.envrc`?

Reply:
this makes sense, but there is a node.env on the server with different parameters... @alanna-zhou thoughts?

Reply:
the reason why we have `.envrc` and `.env` is because `.envrc` is used with `dotenv` to streamline our local development ways of loading in environment variables, and `.env` is strictly used by docker-compose files on the server.